paleofetch.c

```diff
@@ -222,8 +223,27 @@ char *get_resolution() {
 }
 
 char *get_terminal() {
+    Display *display = XOpenDisplay(NULL);
```

Comment:
I think since we open the display in here and in `get_resolution`, I think this could be a global variable alongside `uname_info` and `my_sysinfo` so that we only open and close and do error handling once